core/src/main/resources/lib/audio.fjs

```diff
@@ -33,20 +32,27 @@ export default (componentType: ComponentType) => ({ ophan, dom, viewport }: Serv
       Safari 11+ doesn't recognise our handling of a click event intercepted over a Channel and therefore
       refuses to start playback in response, so we'll use the default addEventListener technique instead
     */
-    a.playPauseButton.addEventListener('click', onPlayPause(a));
-    a.scrubber.addEventListener('input', onTimeSeek(a));
-    a.scrubber.addEventListener('change', onTimeSeek(a));
-    a.player.addEventListener('timeupdate', updateTime(a));
-    a.player.addEventListener('ended', onPlaybackFinished(a));
-
-    observer = onVisible(a);
-    viewport.observe(root, 1, observer);
-
+    audio = a;
+    audio.playPauseButton.addEventListener('click', onPlayPause);
+    audio.scrubber.addEventListener('input', onTimeSeek);
+    audio.scrubber.addEventListener('change', onTimeSeek);
+    audio.player.addEventListener('timeupdate', updateTime);
+    audio.player.addEventListener('ended', onPlaybackFinished);
+    viewport.observe(root, 1, onVisible);
+    
+    recordOphanAudioEvent(a.player.getAttribute('data-media-id') || '', MediaEvents.READY);
```

Comment:
I pulled the READY event because it wasn't doing what it is really intended for. The READY should fire when the media has loaded I think. Is that what this is doing now?

Reply:
It is firing as soon as the player can start playing. This is not strictly true, as no preloading, but the UI is ready to receive interactions. This aligns with frontend.